src/blocks/battery.rs

```diff
@@ -508,7 +508,12 @@ impl Block for Battery {
             self.output.set_icon(match status.as_str() {
                 "Discharging" => "bat_discharging",
```

Comment:
I'm not quite sure I understand what will happen here. Won't these icons only appear if the battery fails to report correctly as charging/discharging? How do you trigger them on your system?

Also, note that `bat_discharging` is the 50% battery icon; perhaps that could be used as well.

Reply:
Oof, so it seems as my battery doesn't manage to properly report the status (I often have "UNK" with the default i3bar for instance) which is why I probably misread the code. I fixed it now, in case of an unknown status it still tries to determine the status now :)